egs/wsj0-mix/create_meta_files.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
Could this follow the same format as in the WHAM recipes? It would make it much easier to switch from one to the other (specially for the wsj0-3mix that is not covered in WHAM)